docs/releases/4.1.txt

```diff
@@ -685,6 +685,10 @@ Miscellaneous
 
 * ``django.contrib.auth.hashers.CryptPasswordHasher`` is deprecated.
 
+* The ability to pass ``nulls_first=False`` or ``nulls_last=False`` to
+  ``Expression.asc()`` and ``Expression.desc()`` methods, and ``OrderBy``
+  expression is deprecated. Use ``None`` instead.
```

Comment:
possibly: 
`and the ``OrderBy`` expression`
or:
`and ``OrderBy`` expressions`
?
(And naturally mirror in the deprecation doc, if changed)

Reply:
Thanks, updated :+1: